manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -467,13 +497,14 @@ func (r *FybrikApplicationReconciler) GetWorkloadCluster(application *api.Fybrik
 }
 
 // NewFybrikApplicationReconciler creates a new reconciler for FybrikApplications
+// A logger is created that logs the standard
 func NewFybrikApplicationReconciler(mgr ctrl.Manager, name string,
 	policyManager connectors.PolicyManager, catalog connectors.DataCatalog, cm multicluster.ClusterLister,
 	provision storage.ProvisionInterface, configEvaluator adminconfig.EvaluatorInterface) *FybrikApplicationReconciler {
 	return &FybrikApplicationReconciler{
 		Client:            mgr.GetClient(),
 		Name:              name,
-		Log:               ctrl.Log.WithName("controllers").WithName(name),
+		Log:               logging.LogInit(logging.CONTROLLER, name, "Coordinator Cluster"), // TODO - get the real cluster name
```

Comment:
You can't know the cluster name at this point. The main process has not finished yet to initialize cache, config, etc. so trying to read a configmap at this point won't work. Cluster can be initialized later, at the first reconcile.

Reply:
Correct.  I removed it completely.  I spoke with @roee88 and we decided not to pass cluster at all.  The logging agent will add the information when it collects the logs.